cookbooks/vankeken_subduction/doc/vankeken_subduction.md

```diff
@@ -0,0 +1,24 @@
+# Van Keken 2008 corner flow recreation
+
+*This section was contributed by Daniel Douglas and Max Rudolph*.
```

Comment:
I think you probably should also credit Cedric.

Reply:
Definitely agree!